src/main/java/com/terraformersmc/campanion/mixin/MixinMobEntity.java

```diff
@@ -16,7 +17,7 @@
 	@Inject(method = "getPreferredEquipmentSlot", at = @At("HEAD"), cancellable = true)
 	private static void onGetPreferredEquipmentSlot(ItemStack stack, CallbackInfoReturnable<EquipmentSlot> info) {
 		Item item = stack.getItem();
-		if (item instanceof BackpackItem) {
+		if (!Curios.isEnabled() && item instanceof BackpackItem) {
```

Comment:
Is there a way to set the preferred slot to the back slot if curious is enabled?

Reply:
If the Curios UI is **open**, then backpacks will auto-equip to the back slot. The onGetPreferredEquipmentSlot is not needed.

If the Curios UI is **not open**, then backpacks will not auto-equip. I'll look at how I can fix that.